juniper_official/routing/check-te-rsvp-interface-errors.rule

```diff
@@ -0,0 +1,487 @@
+                        }
+                        next;
+                    }
+                }
+                term out-path-error-messages {
+                    when {
+                        increasing-at-least-by-value "$out-path-error-messages" {
+                            value 1;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Out path error messages count($out-path-error-messages)increasing on $te-interface";
+                        }
+                        next;
+                    }
+                }
+                term out-reservation-error-messages {
+                    when {
+                        increasing-at-least-by-value "$out-reservation-error-messages" {
+                            value 1;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Out reservation error messages count($out-reservation-error-messages)increasing on $te-interface";
+                        }
+                        next;
+                    }
+                }
+                term is-received-nack {
+                    when {
+                        increasing-at-least-by-value "$received-nack" {
+                            value 1;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Received nack error count($received-nack)increasing on $te-interface";
+                        }
+                        next;
+                    }
+                }
+                term recv-pkt-disabled-intf {
+                    when {
+                        increasing-at-least-by-value "$recv-pkt-disabled-intf" {
+                            value 1;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Recv-pkt-disabled-intf error count($recv-pkt-disabled-intf)increasing on $te-interface";
+                        }
+                        next;
+                    }
+                }
+                term send-failure {
+                    when {
+                        increasing-at-least-by-value "$send-failure" {
+                            value 1;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Send failure error count($send-failure)increasing on $te-interface";
+                        }
+                        next;
```

Comment:
Since all the states are red here except the last one, why do we want to do next?

Reply:
Next is needed so all the terms are processed, else we will have to add seperate triggers